Extensions/xkit_patches.js

```diff
@@ -1180,6 +1181,7 @@ XKit.extensions.xkit_patches = new Object({
 
 				destroy_collapsed: function(id) {
 					$(`.${id}-collapsed`).removeClass(`${id}-collapsed`);
+					$(`.xkit-collapsed`).removeClass(`xkit-collapsed`);
```

Comment:
Okay so realistically no recommended and show originals are unlikely to overlap their uninstall/teardown processes so all of this is fake and doesn't matter but I think this logic is a bit better as the following:
```javascript
$(`.${id}-collapsed`).removeClass(`xkit-collapsed`);
$(`.${id}-collapsed`).removeClass(`${id}-collapsed`);
```
As currently written if show originals calls destroy_collapsed it'll put no recommended's collapsed posts into a weird state.